docs/en/setup/service-agent/java-agent/agent-optional-plugins/Kotlin-Coroutine-plugin.md

```diff
@@ -0,0 +1,30 @@
+# Skywalking with Kotlin coroutine
```

Comment:
As an optional plugin, this doc should be linked from https://github.com/apache/skywalking/blob/master/docs/en/setup/service-agent/java-agent/README.md#optional-plugins